.travis.yml

```diff
@@ -19,6 +19,7 @@ install:
   - ln -s $(pwd) ../hardware/keyboardio/avr/libraries/Kaleidoscope
 script:
   - unset CC
+  - make cpplint BOARD_HARDWARE_PATH=$(pwd)/../hardware
```

Comment:
This is temporary, and we probably should drop it before merging, and roll it into `travis-test` instead. On the other hand, we should probably roll `make -C tests all` into that aswell. So lets keep it here for now, and drop it later, when we update `travis-tests`?